Scenarios/StepDefinition.swift

```diff
@@ -27,8 +27,9 @@ open class StepDefinition: QuickSpec {
 
     return {
       guard let (args, definition) = stepDefinitions.lazy
-        .flatMap({ pattern, function in
-          pattern.firstMatch(in: description).map {
+        .flatMap({ args -> (StepArguments, StepDefinitionFunc)? in
```

Comment:
I think that the style here makes this a bit of an oddball. 

Would you be happy to change this to use a trailing closure?